config/application.rb

```diff
@@ -7,6 +7,7 @@
 require "action_view/railtie"
 require "sprockets/railtie"
 require "rails/test_unit/railtie"
+require "mongoid"
```

Comment:
Is there a reason this line was added?

Reply:
Added for an earlier error that now isn't appearing, removed 👍

Reply:
Looks like it's still here (in a7e583ef798b3702f0e5446ed0b312757f057960).

Reply:
Ah, I see what I did, sorted now 👍